plugins/chat/assets/javascripts/discourse/components/chat-message-info.js

```diff
@@ -64,7 +34,36 @@ export default class ChatMessageInfo extends Component {
     if (user.groupModerator) {
       classes.push("is-category-moderator");
     }
-
+    if (user.new_user) {
```

Comment:
Why do we need these, is it just something that we add elsewhere in core e.g. in posts?

Reply:
yes I added it for parity with post: https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/widgets/poster-name.js#L125